frontend/src/container/GridCardLayout/GridCard/WidgetGraphComponent.tsx

```diff
@@ -273,6 +278,7 @@ function WidgetGraphComponent({
 				onCancel={onToggleModelHandler}
 				width="85%"
 				destroyOnClose
+				className="full-view-modal"
```

Comment:
Please use specific classname, full view modal is used for full view of the widget

Reply:
Changed it ✅